dto.c

```diff
@@ -106,7 +106,7 @@ static uint8_t use_std_lib_calls;
 static enum numa_aware is_numa_aware;
 static size_t dsa_min_size = DTO_DEFAULT_MIN_SIZE;
 static int wait_method = WAIT_YIELD;
-static double cpu_size_fraction;
+static size_t cpu_size_fraction;
```

Comment:
maybe a comment here saying the range (0 to 99?)

Reply:
I've added this one also